src/listing--rating-panel.html

```diff
@@ -9,11 +9,12 @@
 
         <link rel="apple-touch-icon" href="apple-touch-icon.png">
         <!-- Place favicon.ico in the root directory -->
+        <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
 
         <!-- bower:css -->
         <link rel="stylesheet" href="../bower_components/normalize-css/normalize.css" />
         <!-- endbower -->
-        <link rel="stylesheet" href="css/main.css">
+        <link rel="stylesheet" href="css/listing--rating-panel.css">
```

Comment:
Your `listing--rating-panel.scss` file should be `@import`-ed _inside_ `main.scss` to avoid having to link it here. Again, I know you're not the only one doing it this way, so correct it and spread the word!